ansible/roles/nginx/templates/aaad.conf.upstart.j2

```diff
@@ -14,6 +14,8 @@ env SESSION_TOKEN_SECRET_KEY={{ aaadsid_secret_key }}
 env REMEMBER_COOKIE_DURATION_SECONDS={{ aaadsid_rem_cookie_duration_seconds }}
 
 env MESOS_MASTER_URL={{ mesos_leader_ui_url | default('http://127.0.0.1:5050') }}
+env MARATHON_MASTER_URL=http://{{ nginx_listen_on_host|default('127.0.0.1') }}:{{ marathon_http_port }}
```

Comment:
Shouldn't this be the direct one (like `localhost:8080`)?  I suggest you define a var and use that. That will be needed for localmesos anyway.